force-app/main/default/flows/Opportunity_Generate_Deal_Support_Processes.flow-meta.xml

```diff
@@ -727,7 +727,7 @@
         <isInput>false</isInput>
         <isOutput>false</isOutput>
         <value>
-            <stringValue>Advisory</stringValue>
+            <stringValue>Enterprise Advisory</stringValue>
         </value>
```

Comment:
I think this needs to stay "Advisory" as well, but afterwards I think we're all set.

Reply:
Thanks for catching that! This is now reverted as well.